client/src/index.tsx

```diff
@@ -62,11 +64,24 @@ async function start(): Promise<void> {
             faceCountMax: 4000,
         }
         const fabricKernel = new FabricKernel(engine, physics, dimensions)
+        const getFabric = (name: string) => {
```

Comment:
Here, `name` is the construction code, right? If so, better to rename the parameter